dctest-meows-runner/Dockerfile

```
@@ -48,14 +50,18 @@ RUN apt-get update \
             iptables \
             openssh-client \
             sudo \
+            hub \
     && rm -rf /var/lib/apt/lists/* \
     && curl -sSLf https://dl.google.com/go/go${GO_VERSION}.linux-amd64.tar.gz | tar -C /usr/local -xzf - \
     && curl -sfL https://github.com/cybozu-go/placemat/releases/download/v${PLACEMAT_VERSION}/placemat2_${PLACEMAT_VERSION}_amd64.deb -o placemat2_${PLACEMAT_VERSION}_amd64.deb \
     && dpkg -i ./placemat2_${PLACEMAT_VERSION}_amd64.deb \
     && rm ./placemat2_${PLACEMAT_VERSION}_amd64.deb \
     && echo "actions ALL=(ALL) NOPASSWD: ALL" > /etc/sudoers \
-    && adduser --disabled-password --gecos "" --uid 10000 actions 
+    && adduser --disabled-password --gecos "" --uid 10000 actions \
+    && mkdir -p /opt/hostedtoolcache \
+    && chown -R actions:actions /opt/hostedtoolcache \
+    && chown -R actions:actions ${HOME}
 
-COPY bootstrap.sh /root/
+COPY bootstrap.sh ${HOME}
```

This COPY creates a file owned by root.  The mode of the file seems to be copied from the local environment.
If `docker build` is executed in the local copy created under `umask 007`, this COPY creates a file with mode `-rw-rw----` owned by root.  It is unreadable for actions.

```suggestion
COPY --chown=actions:actions bootstrap.sh ${HOME}
```

---

thank you!
I tried it in my hand, and it seems to be readable by actions without the chown option, as follows.

````
actions@a741544b98df:~$ ls -al bootstrap.sh 
-rw-rw-r-- 1 root root 815 Sep 7 02:11 bootstrap.sh
actions@a741544b98df:~$ cat bootstrap.sh 
#! /bin/sh -xe

echo "setup neco and neco-apps"
~~~~
```

However, the owner of `bootstrap.sh` is unintended, so i fixed it.